src/App.vue

```diff
@@ -108,6 +116,29 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
     }
   }
 
+  @Watch('isSoraAccountConnected')
+  private async confirmInviteUserIfConnected(isSoraConnected: boolean): Promise<void> {
+    if (isSoraConnected) {
+      await this.confirmInvititation();
+    }
+  }
+
+  @Watch('storageReferral')
```

Comment:
This modal does not open for me. After adding the option `immediate: true` to this watcher , it began to work